### ISSUE TITLE
Attacking enemies from any angle

### ISSUE THREAD

**Issue by arturaslt** [vis2k/uMMORPG_CE#16]
When fighting enemies, you can attack them from any anggle, litteraly turning your back on the enemy and pressing attack button, will attack them, they should only attack if you face them in front of you.
Using the latest 2.15 version.
And recommended version of unity

**Comment by jon-malave** [comment]
> When fighting enemies, you can attack them from any anggle, litteraly turning your back on the enemy and pressing attack button, will attack them, they should only attack if you face them in front of you.
> Using the latest 2.15 version.
> And recommended version of unity

Yep, I have the same issue... sadly @vis2k will probably never fix this because you know he saved you X amount of hours to help you create your dream MMORPG while many features don't work as expected out the box. No surprise! 

Hopefully you made some progress by now since May 28th on this... maybe we need to create out own uMMORPG unofficial support group since uMMORPG support is shit! feel free to ping me! Hopefully we can figure this out together at some point.
